cmd/imagec/main.go

```diff
@@ -100,12 +108,18 @@ func init() {
 func main() {
 	defer func() {
 		if r := recover(); r != nil {
-			fmt.Fprintf(os.Stderr, string(sf.FormatError(fmt.Errorf("%s : %s", r, debug.Stack()))))
+			_, err := fmt.Fprintf(os.Stderr, string(sf.FormatError(fmt.Errorf("%s : %s", r, debug.Stack()))))
+			if err != nil {
+				//do this to pass security check
```

Comment:
If this is for gas and you want to ignore the error here, you can just call `fmt.Fprintf(...)` above (and don't assign the output to anything), with a `#nosec` on the line above.